worker/user_data.js

```diff
@@ -2,136 +2,14 @@
-                    hasBeenNotified: false
-                },
-                "2021-02-07": {
-                    hasBeenNotified: false
-                },
-                "2021-02-14": {
-                    hasBeenNotified: false
-                },
-                "2021-02-20": {
-                    hasBeenNotified: false
-                },
-                "2021-02-21": {
-                    hasBeenNotified: false
-                },
-                "2021-02-27": {
-                    hasBeenNotified: false
-                },
-                "2021-02-28": {
-                    hasBeenNotified: false
-                }
-            },
-            "taos": {
-                "2021-03-13": {
-                    hasBeenNotified: false
-                },
-                "2021-03-14": {
-                    hasBeenNotified: false
-                },
-                "2021-03-20": {
-                    hasBeenNotified: false
-                },
-                "2021-03-21": {
-                    hasBeenNotified: false
-                },
-                "2021-03-27": {
-                    hasBeenNotified: false
-                },
-                "2021-03-28": {
-                    hasBeenNotified: false
-                },
-            }
-        }
-    },
-    // "louis": {
-    //     passType: "base",
-    //     telegram_id: process.env.LOUIS_TELEGRAM_CHAT_ID,
-    //     desiredSkiDays: {
-    //         "winterpark": {
-    //             "2021-02-13": {
-    //                 hasBeenNotified: false
-    //             },
-    //         },
-    //     }
-    // },
-    "mark": {
-        passType: "base",
-        telegram_id: process.env.MARK_TELEGRAM_CHAT_ID,
-        desiredSkiDays: {
-            "winterpark": {
-                "2021-02-13": {
-                    hasBeenNotified: false
-                },
-            },
-            "abasin": {
-                "2021-01-23": {
-                    hasBeenNotified: false
-                },
-                "2021-01-24": {
-                    hasBeenNotified: false
-                },
-                "2021-01-10": {
-                    hasBeenNotified: false
-                },
-                "2021-01-09": {
-                    hasBeenNotified: false
-                },
-                "2021-01-16": {
-                    hasBeenNotified: false
-                },
-                "2021-01-17": {
-                    hasBeenNotified: false
-                },
-            },
-            "taos": {
-                "2021-03-06": {
-                    hasBeenNotified: false
-                },
-                "2021-03-07": {
-                    hasBeenNotified: false
-                },
-                "2021-01-04": {
-                    hasBeenNotified: false
-                },
-            }
-        }
-    }
+    "719487075:pass_type": 'base',
+    "719487075:winterpark":"[2021-02-02, 2021-02-03,2021-02-04, 2021-02-05]",
```

Comment:
is this in the correct format? I took all the dates we had in master and converted them to:
"chat_id:resort_name":["requestedDate", "requestedDate"] but you have it as:
"chat_id:resort_name":"[requestedDate, requestedDate]".

here is the link to the user_dates I did with all the dates we had in master - https://github.com/jbrennan414/ski-scraper/blob/add_through_telegram/worker/user_data.js